themes/src/main/resources/theme/base/login/login.ftl

```diff
@@ -6,7 +6,7 @@
     <div id="kc-form">
       <div id="kc-form-wrapper">
         <#if realm.password>
-            <form id="kc-form-login" onsubmit="login.disabled = true; return true;" action="${url.loginAction}" method="post">
+            <form id="kc-form-login" action="${url.loginAction}" method="post">
```

Comment:
Similar code for this exists in [`login-password.ftl`](https://github.com/keycloak/keycloak/blob/84e763b472cd6a3f4550e962d694cd0016660c24/themes/src/main/resources/theme/base/login/login-password.ftl#L8) and  [`login-username.ftl`](https://github.com/keycloak/keycloak/blob/84e763b472cd6a3f4550e962d694cd0016660c24/themes/src/main/resources/theme/base/login/login-username.ftl#L9) which should be removed there as well.

Reply:
Done